modules/gitops-enable-traffic-management-and-metric-plugins-in-argo-rollouts.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/configuring_traffic_management_and_metric_plugins_in_argo_rollouts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enable-traffic-management-and-metric-plugins-in-argo-rollouts_{context}"]
+= Enabling Traffic management and Metric plugins in Argo Rollouts
+
+To enable traffic management and metric plugins in Argo Rollouts, complete the following steps.
+
+.Procedure 
+
+. Log in to the {OCP} web console as a cluster administrator. 
+
+. In the *Administrator* perspective, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to create and configure a `RolloutManager` custom resource (CR) from the *Project* drop-down menu.
+
+. Select *{gitops-title}* from the installed operators.
+
+. In the *Details* tab, under the *Provided APIs* section, click *Create instance* in the *RolloutManager* pane.
+
+. On the *Create RolloutManager* page, select the *YAML view* and edit the YAML.
++
+.Example for adding the traffic management and metric plugins configuration in the `RolloutManager` CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: argo-rollouts
+  labels:
+    example: basic
+spec:
+  plugins:
+    trafficManagement:
+      - name: argoproj-labs/gatewayAPI #<1>
+        location: https://github.com/sample-metric-plugin #<2>
+    metric:
+      - name: argoproj-labs/sample-prometheus #<3>
+        location: https://github.com/sample-trafficrouter-plugin #<4>
+        sha256: dac10cbf57633c9832a17f8c27d2ca34aa97dd3d
```

Comment:
Let's add a `#<5>` to the `sha256:` line:
`            sha256: dac10cbf57633c9832a17f8c27d2ca34aa97dd3d  # <5>`

And then below
```
<5> Optional: Specifies the SHA256 signature of the plugin binary, which is downloaded and installed by the Rollouts controller.
```

This makes it explicit what sha256 is for, and that it is optional.

Reply:
Thanks for the great feedback. Fixed.